src/api/db/structure.sql

```diff
@@ -1177,14 +1166,14 @@ CREATE TABLE `status_messages` (
 
 CREATE TABLE `status_reports` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `uuid` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-  `checkable_type` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `uuid` varchar(255) DEFAULT NULL,
+  `checkable_type` varchar(191) DEFAULT NULL,
   `checkable_id` int(11) DEFAULT NULL,
   `created_at` datetime NOT NULL,
   `updated_at` datetime NOT NULL,
   PRIMARY KEY (`id`),
   KEY `index_status_reports_on_checkable_type_and_checkable_id` (`checkable_type`,`checkable_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
```

Comment:
I think the collation should stay like this. It should have been like that in the first place. Somebody is running an old configuration.

Reply:
@dmarcoux, thanks